fastparquet/writer.py

```diff
@@ -438,14 +443,18 @@ def write_column(f, data, selement, encoding='PLAIN', compression=None):
 
 def make_row_group(f, data, schema, file_path=None, compression=None,
                    encoding='PLAIN'):
+    """ Make a single row group of a Parquet file """
     rows = len(data)
-    rg = parquet_thrift.RowGroup(num_rows=rows, total_byte_size=0,
-                                 columns=[])
+    rg = parquet_thrift.RowGroup(num_rows=rows, total_byte_size=0, columns=[])
 
     for column in schema:
         if column.type is not None:
+            if isinstance(compression, dict):
+                comp = compression.get(column.name, None)
+            else:
+                comp = compression
             chunk = write_column(f, data[column.name], column,
-                                 compression=compression, encoding=encoding)
+                                 compression=comp, encoding=encoding)
```

Comment:
Thoughts on passing around some "compression parameters"? Or perhaps having global variables in the compression module which can be changed. I am thinking of the slow default in gzip to go for maximal compression.